src/components/content/monitor/MonitorChart.tsx

```diff
@@ -3,50 +3,326 @@
+            stopFetchOnlyLastKnownMonitorMetricDataTimer();
+            getTipInfo('', false, 'error', 'An exception occurred in the request metric timer.', true);
+        }
+    };
+
+    const startFetchMonitorMetricDataTimer = () => {
+        try {
+            refreshMonitorMetrics.current = window.setInterval(() => {
+                getMetricResponse(serviceId, activeMonitorMetricType as Metric.monitorResourceType, false);
+            }, fetchMonitorMetricDataTimeInterval);
+        } catch (error) {
+            setIsAutoRefresh(false);
+            setMonitorMetricsQueue([]);
+            stopFetchMonitorMetricDataTimer();
+            getTipInfo('', false, 'error', 'An exception occurred in the request metric timer.', true);
+        }
+    };
+
+    const stopFetchOnlyLastKnownMonitorMetricDataTimer = () => {
+        if (refreshOnlyLastKnownMonitorMetrics.current) {
+            clearInterval(refreshOnlyLastKnownMonitorMetrics.current);
+            refreshOnlyLastKnownMonitorMetrics.current = null;
+        }
+    };
+
+    const stopFetchMonitorMetricDataTimer = () => {
+        if (refreshMonitorMetrics.current) {
+            clearInterval(refreshMonitorMetrics.current);
+            refreshMonitorMetrics.current = null;
+        }
+    };
+
+    const getOnlyLastKnownMetricResponse = (selectedServiceId: string, onlyLastKnownMetric: boolean) => {
+        void MonitorService.getMetricsByServiceId(
+            selectedServiceId,
+            undefined,
+            undefined,
+            undefined,
+            undefined,
+            onlyLastKnownMetric
+        )
+            .then((rsp: Metric[]) => {
+                if (rsp.length > 0) {
+                    getTipInfo(serviceId, false, undefined, '', true);
+                    setOnlyLastKnownMonitorMetricsQueue((prevQueue: Metric[][] | undefined) => {
+                        let newQueue: Metric[][];
+
+                        if (prevQueue) {
+                            newQueue = [...prevQueue, rsp];
+                            if (newQueue.length > monitorMetricQueueSize) {
+                                newQueue.shift();
+                            }
+                        } else {
+                            newQueue = [rsp];
+                        }
+
+                        return newQueue;
+                    });
+                } else {
+                    setIsAutoRefresh(false);
+                    setOnlyLastKnownMonitorMetricsQueue([]);
+                    stopFetchOnlyLastKnownMonitorMetricDataTimer();
+                    getTipInfo('', false, 'success', 'No metrics found for the selected service.', true);
+                }
+            })
+            .catch(() => {
+                setIsAutoRefresh(false);
+                setOnlyLastKnownMonitorMetricsQueue([]);
+                stopFetchOnlyLastKnownMonitorMetricDataTimer();
+                getTipInfo('', false, 'error', 'Error while fetching metrics data.', true);
```

Comment:
for all the catch blocks which we have for API calls, we must extract the error out of it and display it in the alert description. 

Example - https://github.com/eclipse-xpanse/xpanse-ui/blob/f845b6938c95e8c84ad2a410ea9d6592a634e59a/src/components/content/register/registerService.ts#L27

Reply:
fixed.